src/main/java/com/twilio/oai/TwilioHelperSDK.java

```diff
@@ -0,0 +1,16 @@
+package com.twilio.oai;
+
+public enum TwilioHelperSDK {
+    TWILIO_PHP("twilio-php");
```

Comment:
We're defining the names of other helpers in the constants file. Can these be unified?